sync-gateway-config.json

```diff
@@ -12,38 +12,46 @@ function(doc, oldDoc) {
   // NOTE this function is the same across the iOS, Android, and PhoneGap versions.
   if (doc.type == "task") {
     if (!doc.list_id) {
-      throw({forbidden : "items must have a list_id"})
+      throw({forbidden : "items must have a list_id"});
     }
     channel("list-"+doc.list_id);
-  } else if (doc.type == "list") {
-    channel("list-"+doc._id);
-    if (!doc.owner) {
-      throw({forbidden : "list must have an owner"})
+  } else if (doc.type == "list" || (doc._deleted && oldDoc && oldDoc.type == "list")) {
+    // Make sure that only the owner of the list can update the list:
+    var owner = oldDoc ? oldDoc.owner : doc.owner;
```

Comment:
It looks like this lets the oldDoc.owner change doc.owner to someone other than themselves?  Is that intentional?